README.md

```diff
@@ -9,7 +9,13 @@ An extension for Gnome-Shell to show Hijri date/calendar
 * Create a directory in `~/.local/share/gnome-shell/extensions/` and name it `HijriCalendar@oxygenws.com`
 * Copy all files in the repository to the above directory (`~/.local/share/gnome-shell/extensions/HijriCalendar@oxygenws.com/`)
 * Restart Gnome-shell. (ALT+F2, r, Enter)
-
+Or run this commands:
+```bash
+mkdir -p ~/.local/share/gnome-shell/extensions/HijriCalendar@oxygenws.com
+git clone git@github.com:omid/Hijri-Calendar-for-Gnome-Shell.git ~/.local/share/gnome-shell/extensions/HijriCalendar@oxygenws.com
+gnome-shell --replace
```

Comment:
By running this command, gnome-shell will attach to the terminal which you ran this command.
People may close the terminal and face problems.

Reply:
We can remove the last line and let the replace be done manually

Reply:
Yes and no :D

There are two ways to run it...
1. run it in ALT+F2
2. run "gnome-shell --replace &" to run it in bg and then "disown"

And also you can create a Makefile for this and put the whole logic in it